eng/build-helpers.psm1

```diff
@@ -57,16 +57,58 @@ function Invoke-Main {
 
     try {
         &$MainBlock
-        Write-Host
-        Write-Host "Build Succeeded" -fore GREEN
+        Write-NewLine
+        Write-Success "Build Succeeded"
         exit 0
     } catch [Exception] {
-        Write-Host
+        Write-NewLine
         Write-Error $_.Exception.Message
-        Write-Host
+        Write-NewLine
         Write-Error "Build Failed"
         exit 1
     }
 }
 
-Export-ModuleMember -Function *
\ No newline at end of file
+#Output message using the Cyan text color
+function Write-Command($message){
```

Comment:
No change requested right now. A tip for future reference: (a) the reader can see that it is for Cyan, so no need for the comment 😄 and (b) if we decide to change the color in the future, we'll have to change in two places - the comment and the actual code - instead of just one.

Reply:
I changed my mind: good practice and learning opportunity! Pleas go update the comment. Rather than saying precisely what it does, explain why it should be used: "Display a command and arguments on the console."